jquery.flot.composeImages.js

```diff
@@ -150,13 +150,33 @@ temporary images load their data.
     }
 
     function copySVGToImgSafari(svg, img) {
```

Comment:
I could not get rid of this method an make everything use 'copySVGToImgMostBrowsers'.

In Safari setting img.src does not trigger a call to the 'onload' event which is expected to resolve the Promise that is waiting for this image.  For this reason we have this special behavior for Safari, where we have to generate data URLs.